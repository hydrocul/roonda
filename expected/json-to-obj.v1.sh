#!/bin/sh

#################################################
# data.js:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/data.js
[["abc", "Hey", 10],
 ["def", "Hello", 30],
 ["ghi", "Hello", 50],
 ["jk", "Hello   World!", 100]]
END_OF_ROONDA_SOURCE_FILE
#################################################

cat $ROONDA_TMP_PATH/data.js | $ROONDA_SELF_PATH --v1 --json-to-perl-obj
cat $ROONDA_TMP_PATH/data.js | $ROONDA_SELF_PATH --v1 --json-to-ruby-obj
cat $ROONDA_TMP_PATH/data.js | $ROONDA_SELF_PATH --v1 --json-to-python2-obj
cat $ROONDA_TMP_PATH/data.js | $ROONDA_SELF_PATH --v1 --json-to-python3-obj
cat $ROONDA_TMP_PATH/data.js | $ROONDA_SELF_PATH --v1 --json-to-php-obj
