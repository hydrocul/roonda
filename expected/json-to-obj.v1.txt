[["abc","Hey",10],["def","Hello",30],["ghi","Hello",50],["jk","Hello   World!",100]]
[["abc","Hey",10],["def","Hello",30],["ghi","Hello",50],["jk","Hello   World!",100]]
[["abc","Hey",10],["def","Hello",30],["ghi","Hello",50],["jk","Hello   World!",100]]
[["abc","Hey",10],["def","Hello",30],["ghi","Hello",50],["jk","Hello   World!",100]]
array(array("abc","Hey",10),array("def","Hello",30),array("ghi","Hello",50),array("jk","Hello   World!",100))
